content/projects/data-science-specific/statistical-thinking/_index.md

```diff
@@ -42,38 +42,53 @@ Complete the DataCamp courses [Statistical Thinking in Python Part 1](https://ww
   - Do you notice any unusual/extreme values that do not fit the general trend? If you see any unusual values, briefly describe them (Who are they? In what way are they different?).
 
-11. Since neither forested land area nor protected area is normally distributed, we will need to log transform these variables in order to calculate a correlation coefficient. Log transform the variables and show the transformed distributions in a histogram.
+14. Since neither forested land area nor protected area is normally distributed, we will need to log transform these variables in order to calculate a correlation coefficient. Log transform the variables and show the transformed distributions in a histogram. You need to create 2 DataFrames:
+  - `log_transformed_land_area_covered_2000_df` with column names `Country` and `log_transformed_forested_land_area_value`
+  - `log_transformed_protected_area_2000_df` with column names `Country` and `log_transformed_protected_area_value`.
 
-12. Using the `pearsonr` function from the `scipy.stats` module, calculate the Pearson correlation coefficient (and its corresponding p value) to determine the nature of the relationship between proportion of land area covered by forest and the percentage of area protected (as measured in 1990 and log transformed). See `help(pearsonr)` for help on this function.
+15. We'd like to determine the nature of the relationship between proportion of land area covered by forest and the percentage of area protected (as measured in 1990 and log transformed). Start with stating the "null" and "alternative" hypothesis for this enquiry. Using the `pearsonr` function from the `scipy.stats` module, calculate the Pearson correlation coefficient and its corresponding p value. Save this answer in a variable called `pearson_correlation_coefficient_1990`. The p value here should be saved in a variable called `pearson_p_value_1990`. See `help(pearsonr)` for help on this function.
 
   - Interpret the size and direction of the correlation statistic.
   - Is the relationship statistically significant? Report the appropriate statistic(s) to support your answer.
 
-13. Calculate the Spearman Rank-Order Correlation Coefficient. This test only looks at the _order_ of the categories, not the _values_. The Spearman Rank-Order Coefficient is therefore not influenced by non-normality of variables or outliers. How do the results of this test compare the results of the Pearson's correlation?
+16. Calculate the Spearman Rank-Order Correlation Coefficient. This test only looks at the _order_ of the categories, not the _values_. Save this answer in a variable called `spearman_correlation_coefficient_1990`. The p value here should be saved in a variable called `spearman_p_value_1990`. The Spearman Rank-Order Coefficient is therefore not influenced by non-normality of variables or outliers. How do the results of this test compare with the results of the Pearson's correlation?
```

Comment:
this does also need a slight rewording to make better sense - 

"Calculate the Spearman Rank-Order Correlation Coefficient. Save this answer in a variable called `spearman_correlation_coefficient_1990`. The p value here should be saved in a variable called `spearman_p_value_1990`. This test only looks at the _order_ of the categories, not the _values_. The Spearman Rank-Order Coefficient is therefore not influenced by non-normality of variables or outliers. How do the results of this test compare with the results of the Pearson's correlation?"